source/includes/_admin.md

```diff
@@ -109,6 +109,11 @@ curl -v \
     -H "X-Killbill-ApiSecret: lazar" \
     -H "Content-Type: application/json" \
     -H "X-Killbill-CreatedBy: demo" \
+    -d '{
```

Comment:
Would it make sense to provide more info on what these states are?

Reply:
Might also help to include the `AdminPayment` resource object in the docs (Similar to [Invoice Resource](https://killbill.github.io/slate/?java#invoice-invoice-resource))